iothub/device/src/IotHubConnection.cs

```diff
@@ -1,452 +1,326 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.Azure.Devices.Client
+namespace Microsoft.Azure.Devices
```

Comment:
I don't think this is correct: looks like a copy of the ServiceClient connection string parser?